src/main.rs

```diff
@@ -58,7 +64,7 @@ use std::panic;
 use std::process;
 
 fn main() {
-    log::init();
```

Comment:
I'm not entirely happy with needing to expose the guard to `main`, but I think getting rid of the custom logging code is enough of a win.